auth0/v3/management/hooks.py

```diff
@@ -0,0 +1,188 @@
+from .rest import RestClient
+
+
+class Hooks(object):
+
+    """Hooks endpoint implementation.
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+
+        timeout (float or tuple, optional): Change the requests
+            connect and read timeout. Pass a tuple to specify
+            both values separately or a float to set both to it.
+            (defaults to 5.0 for both)
+    """
+
+    def __init__(self, domain, token, telemetry=True, timeout=5.0):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry, timeout=timeout)
+
+    def _url(self, id=None):
+        url = "https://{}/api/v2/hooks".format(self.domain)
+        if id is not None:
+            return "{}/{}".format(url, id)
+        return url
+
+    def all(
+        self,
+        enabled=True,
+        fields=None,
+        include_fields=True,
+        page=None,
+        per_page=None,
+        include_totals=False,
+    ):
+        """Retrieves a list of all hooks.
+
+        Args:
+            enabled (bool, optional): If provided, retrieves hooks that match
+                the value, otherwise all hooks are retrieved.
+
+            fields (list, optional): A list of fields to include or exclude
+                (depending on include_fields) from the result, empty to
+                retrieve all fields.
+
+            include_fields (bool, optional): True if the fields specified are
+                to be included in the result, False otherwise
+                (defaults to true).
+
+            page (int, optional): The result's page number (zero based).
+
+            per_page (int, optional): The amount of entries per page.
+
+            include_totals (bool, optional): True if the query summary is
+                to be included in the result, False otherwise.
+
+        See: https://auth0.com/docs/api/management/v2#!/Hooks/get_hooks
+        """
+
+        params = {
+            "fields": fields and ",".join(fields) or None,
+            "include_fields": str(include_fields).lower(),
+            "page": page,
+            "per_page": per_page,
+            "include_totals": str(include_totals).lower(),
+        }
+
+        # since the default is True, this is here to disable the filter
+        if enabled is not None:
+            params["enabled"] = str(enabled).lower()
+
+        return self.client.get(self._url(), params=params)
+
+    def create(self, body):
+        """Creates a new Hook.
+
+        Args:
+            body (dict): Attributes for the newly created hook,
+                See: https://auth0.com/docs/api/v2#!/Hooks/post_hooks
+        """
+        return self.client.post(self._url(), data=body)
+
+    def get(self, id, fields=None, include_fields=True):
```

Comment:
I don't see any property other than `id` documented here https://auth0.com/docs/api/management/v2#!/Hooks/get_hooks_by_id.

Please, remove fields and include_fields.

Reply:
Thanks for the review. Those fields are indeed not part of the documented fields but the descriptions states "Accepts a list of fields to include or exclude in the result." That's why I included them (I didn't test yet if they work or not). 
I can remove them, but I also suggest to remove this comment from the documentation.

Reply:
I tested, and those `fields` and `include_fields`actually work for this API. So I would suggest fixing the doc (if that might take too long, I can remove them from the PR so they can be added later, just let me know).

Reply:
I could do the PR to fix the docs but I'd first like to check with the Hooks team why these properties are not documented (there might be a reason). 

We can hold off merging this PR until that response comes back (might take ~10 days), or if you want, you can remove these properties for now and if the team later says they are valid, we can update the docs and this SDK on a separate PR. 

How does that sound @guillp?

Reply:
No worries, I don't mind testing for a few days until you sort it out internally.

Reply:
@guillp I got confirmation that for `GET Hooks/{id}` the `fields` argument is supported. What they didn't specify is about the `include_fields` argument. I assume it is, since both relate. Can you run a quick request and test that for me, please?

For secrets, there's no support for `fields`. So good that you removed them. I think we can get this merged and released today if you're around. Please, do rebase the branch since I merged a few PRs a few hours ago.

Reply:
Update from the team: `include_fields` is **not** supported on neither of these endpoints. Let's keep the `fields` arg only for this endpoint. 
Regarding the tests, I retried the build and it passed 👍